### ISSUE TITLE
Annotation for some genome contains a coding region where there is no transcript

### ISSUE THREAD

**Issue by soi** [weberlab-hhu/GeenuFF#79]
Nine genomes training result: 

+Intergenic+-----------+--------+----------+
|          | Precision | Recall | F1-Score |
+----------+-----------+--------+----------+
| cds 0    | 1.0000    | 0.9790 | 0.9894   |
| cds 1    | 0.0000    | 0.0193 | 0.0000   |
| intron 0 | 1.0000    | 0.9872 | 0.9935   |
| intron 1 | 0.0000    | 0.0000 | 0.0000   |
|          |           |        |          |
| total 0  | 1.0000    | 0.9767 | 0.9882   |
| total 1  | 0.0000    | 0.0193 | 0.0000   |
+----------+-----------+--------+----------+

**Comment by soi** [comment]
52947 (array([1008, 1009, 1010, ..., 2168, 2169, 2170]),) b'Mpolymorpha' b'scaffold_3855' [   0 2285]

**Comment by alisandra** [comment]
So this was caused by trying to enter features regardless of strand, which in "normal cases" did nothing, as for an array x, x[n:m] selects nothing when m < n. Except in the edge case of m=-1, then it selects until the end of the array. Good we caught it here, because this probably affected more than just the cases where we saw CDS without Transcript... This has been patched in the branch "cds_wo_transcript", but it seems to have broken two tests. Tomorrow.
